src/site/shared/containers/Header/Left/index.tsx

```diff
@@ -40,26 +43,28 @@ const _HeaderLeft = ({ isLocked, isSaved, isLoggedIn, isLoading, circuitName, he
             <button className="header__left__lock"
                     title="Lock/Unlock Editing"
                     onClick={() => toggleLock()}>
-                <img src="img/icons/lock_open.svg" className={isLocked ? "hide" : ""} alt="Icon for unlocked lock" />
-                <img src="img/icons/lock.svg"      className={isLocked ? "" : "hide"} alt="Icon for lock" />
+                <img src="img/icons/lock_open.svg" className={isLocked ? "hide" : ""} alt="Icon for unlocked lock"/>
+                <img src="img/icons/lock.svg" className={isLocked ? "" : "hide"} alt="Icon for lock"/>
             </button>
         </div>
         <div>
             <input title="Circuit Name" type="text"
                    value={circuitName}
                    placeholder="Untitled Circuit*"
                    onChange={(s) => setCircuitName(s.target.value)}
-                   alt="Name of project" />
+                   alt="Name of project"/>
         </div>
         <div>
-            <button className={`header__left__save ${isSaved || !isLoggedIn ? "invisible" : ""}`}
-                    title="Save the circuit remotely"
-                    disabled={isLoading}
-                    onClick={() => helpers.SaveCircuitRemote() }>Save</button>
+            <button className={`header__left__save ${isSaved ? "invisible" : ""}`}
+                    title={isLoggedIn?"Save the circuit remotely":"Log In to save the circuit"}
+                    disabled={isLoading || !isLoggedIn}
+                    onClick={isLoggedIn ? () => helpers.SaveCircuitRemote() : null}>Save
+            </button>
         </div>
         <div className="header__left__saving__icons">
-            <img src="img/icons/error.svg" className={error ? "" : "hide"} title={`Error occured while saving: ${error}`} alt="Icon when save failed" />
-            <span className={isLoading ? "" : "hide"} title="Saving..."></span>
+            <img src="img/icons/error.svg" className={error ? "" : "hide"}
+                 title={`Error occured while saving: ${error}`} alt="Icon when save failed"/>
+            <span className={isLoading ? "" : "hide"} title="Saving..."/>
```

Comment:
This is a change completely unrelated to #641 , we need to discuss this before just randomly deciding you want to make it